src/app/components/AdminNavbar.js

```diff
@@ -1,29 +1,73 @@
 "use client";
-import Nav from 'react-bootstrap/Nav';
+
+import { Container, Nav, Navbar, Button } from "react-bootstrap";
+import { useRouter } from "next/navigation";
+import { logout } from "../../../utils/supabase/actions";
 import { FaGithub } from "react-icons/fa";
-import Link from "next/link"
-import { useParams } from 'next/navigation';
+import { PiSunBold, PiCalendarDuotone, PiQuestionMarkFill } from "react-icons/pi";
 
 const AdminNavbar = () => {
-  const { _id } = useParams();
+  const router = useRouter();
 
   return (
-    <Nav fill variant="pills" defaultActiveKey={`/admin/${_id}/profile`}>
-    <div>Admin Navbar</div>
-    <Nav.Item>
-    <Nav.Link href={`/admin/${_id}/profile`}>Profile</Nav.Link>
-    </Nav.Item>
-     <Nav.Item>
-      <Nav.Link href={`/admin/${_id}/schedule`}>Schedule</Nav.Link>
-    </Nav.Item>
-    <Nav.Item>
-      <Nav.Link href="/about">About Us</Nav.Link>
-    </Nav.Item>
-    <Link href="https://github.com/chingu-voyages/v52-tier3-team-38" target="_blank">
-    <FaGithub />
-    </Link>
-    </Nav>
-  )
-}
+    <Navbar expand="lg" className="bg-body-tertiary">
+      <Container>
+        {/* Brand - stays the same on all views */}
+        <Navbar.Brand href="/" className="fs-4">
+          <PiSunBold className="me-2 fs-3" />Solarize
+        </Navbar.Brand>
+
+        <Navbar.Toggle aria-controls="basic-navbar-nav" />
+
+        <Navbar.Collapse id="basic-navbar-nav">
+          {/* Main navigation items */}
+          <Nav className="ms-auto me-3">
+            {/* Desktop view: Horizontal list without icons */}
+            <div className="d-none d-lg-flex">
+              <Nav.Link href="/admin" className="px-3">Appointments</Nav.Link>
```

Comment:
It links to a page that is not used. The href should be refactored to link to the list view page.

Reply:
Thank you so much for the clarification! 

I spent some time last night wondering if I should delete that or not!